apps/cookbook/src/app/layout-recipes/grid-layout/grid-layout-multiple-containers/grid-layout-multiple-containers.component.ts

```diff
@@ -0,0 +1,14 @@
+import { Component } from '@angular/core';
+declare var require: any;
```

Comment:
Have you considered instead to add "node" under compilerOptions types in [tsconfig.app.json](https://github.com/kirbydesign/designsystem/blob/2d8dc84753c9f3b5ae6f73baa034cdd51fe24e36/apps/cookbook/tsconfig.app.json) as we already have the node types installed in the project?

Reply:
See https://github.com/kirbydesign/designsystem/pull/1712#discussion_r695553805